applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,157 @@
 # Imaging AI Whole Body Segmentation
 
-This application uses MONAI re-trained TotalSegmentator model to segment 104 body parts from a DICOM series of a CT scan. It is implemented using Holohub DICOM processing operators and PyTorch inference operators.
+This application demonstrates the use of the medical imaging operators to build and package an application that parses DICOM images and performs inference using a MONAI model (TotalSegmentator).
 
-The input is a DICOM CT Series, and the segmentation results are saved as DICOM Segmentation in Part10 storage format, as well as in NIfTI format. The workflow is summarized below,
-- load DICOM studies
-- select series with application defined rules
-- convert DICOM pixel data to 3D volume image
-- use MONAI SDK to transform input/output and perform inference
-- write results as DICOM Segmentation OID instance, re-using study level metadata from the original DICOM study so that the new instance and series can be associated with the original study
+![3D Volume Rendering](resources/segments_3D.png)  
+_Fig. 1: 3D volume rendering of segmentation results in NIfTI format_
 
-The following is the screenshot of the 3D volume rendering of the segmentation results in NIfTI format.
+## Overview
 
-<img src="resources/segments_3D.png" alt="isolated" width="800"/>
+This application uses a MONAI re-trained TotalSegmentator model to segment 104 body parts from a DICOM CT series. It is implemented using Holohub DICOM processing operators and PyTorch inference operators.
 
-The following is the screenshot of a slice of the segmentation saved in DICOM segmentation instance (without color coding the segments).
+The input is a DICOM CT series, and the segmentation results are saved as both DICOM Segmentation (Part10 storage format) and NIfTI format. The workflow includes:
 
-<img src="resources/segments_DICOM_slice.png" alt="isolated" width="800"/>
+- Loading DICOM studies
+- Selecting series with application-defined rules
+- Converting DICOM pixel data to a 3D volume image
+- Using MONAI SDK to transform input/output and perform inference
+- Writing results as a DICOM Segmentation OID instance, re-using study-level metadata from the original DICOM study
+
+The segmentation results are saved in both DICOM Segmentation format (Part10 storage) and NIfTI format for visualization and further analysis.
```

Comment:
Duplication of the second sentence on line 12.

Reply:
Thanks for catching it. Removed the duplicate wording.